runtime/src/chplsys.c

```diff
@@ -52,6 +52,9 @@
 #include <errno.h>
 #include <unistd.h>
 
+#include <signal.h>
```

Comment:
I'm not seeing the need for more `#ifdef` linux constructs.  None of these are in such a construct now.  Wouldn't `#include <signal.h>` go directly before `#include <stdio.h>`, `#include <sys/mman.h>` go directly after `#include <string.h>`, and `#include "chpl-align.h"` go before `#include "chpl-mem.h"`?